seedDB/gene_update_dec_16_2020.md

```diff
@@ -0,0 +1,1142 @@
+# Gene Table Updates Jan 2021
+
+We recently updated our gene tables (main and alias) in cBioPortal public database, and seed database (Download here "put link to new seed DB"), based on the latest [HGNC](https://www.genenames.org/download/statistics-and-files/) with small modifications (see details below). The step-by-step process for constructing the tables, along with the scripts for automating the process can be find here (link to scripts / readme in importer). 
+
+1. HGNC table used: "Complete dataset download links" -> "Complete HGNC approved dataset" (miRNA genes excluded)
+
+2. To reduce loss in data availability, some extra genes are attached. Complete lists [here](#gene_supp).
+
+3. All data files are updated accordingly
+  - gene entries updated: complete lists [here](#gene_update).
+  - eliminate duplciate symbol <> entrez_ID mapping: complete lists [here](#duplicate_mapping_removal)
+  - For local instances, the process for updating data files can be automated by running this script: https://github.com/cBioPortal/datahub-study-curation-tools/pull/29
```

Comment:
Will the link be updated once the PR is merged?

Reply:
@jjgao Yes these links will be updated -- the texts there now are just place holders.